# the following code will raise a ValueError as the value passed to the function int is not a valid integer
